doc/playbook.rst

```diff
@@ -127,6 +127,15 @@ It accepts messages and gateways them to Cloud Run.
 tester
 ======
 
+Run the tester either on a local machine, or in Cloud Shell.  
+For the latter, install the protype code and the GCP pubsub client:
```

Comment:
Did you mean "for the former"? It seems strange that you'd have to do these steps (especially installing `google-cloud-pubsub`) on Google's own system, whereas they make sense on a local machine.

Reply:
Never mind, these instructions are indeed for the Shell.